manim/config/config_utils.py

```diff
@@ -147,9 +147,7 @@ def _parse_file_writer_config(config_parser, args):
     }
 
     # For internal use (no CLI flag)
-    fw_config["skip_animations"] = any(
-        [fw_config["save_last_frame"], fw_config["from_animation_number"]]
-    )
+    fw_config["skip_animations"] = fw_config["save_last_frame"]
```

Comment:
Question: can you explain why this change is necessary?

Reply:
Because, setting `skip_animation` to True here because of `from_animation_number` here does not make any sense, as skip_animation is set in consequence of this flag already in `update_skipping_status`, 

Furthermore, (this is a particular case) when one is trying to start from the Animation 0 with `-n 0,2` manim will skip the first animation as `skip_animations` would have been set to True here.